lms/djangoapps/branding/views.py

```diff
@@ -102,3 +103,7 @@ def courses(request):
     #  we do not expect this case to be reached in cases where
     #  marketing is enabled or the courses are not browsable
     return courseware.views.courses(request)
+
+
+def footer(request):
```

Comment:
Please add a feature flag (`ConfigurationModel`) to control whether the branding API is enabled.  If it's not enabled, then this end-point should return a 404.